src/status_im/search/core.cljs

```diff
@@ -1,5 +1,17 @@
 (ns status-im.search.core
   (:require [status-im.utils.fx :as fx]))
 
-(fx/defn filter-changed [cofx search-filter]
+(fx/defn filter-changed
```

Comment:
@yenda maybe here and in other places could make sense to rename `filter-changed` and all other related events\keys to `chat-filtered-changed`. So we have consistency across all search filter stuff. WDYT?

Reply:
@yenda i did that :)